library/src/scala/util/FromDigits.scala

```diff
@@ -2,9 +2,12 @@ package scala.util
 import scala.math.{BigInt}
 import quoted._
 import annotation.internal.sharable
+import annotation.experimental
+
 
 /** A type class for types that admit numeric literals.
  */
+@experimental
```

Comment:
If we don't enforce it in 3.0.0 then we shouldn't mark code with it in 3.0.0 (because we can't actually change these APIs after 3.0.0 is out without potentially breaking code built against 3.0.0)